### PR TITLE
fix(dependencies): fix package vulnerabilities

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api",
-  "version": "0.16.1",
+  "version": "0.16.2",
```
Comment (RobinKnipe) [inline package.json:3]: You'll also need to make a similar change to `.drone.yml`

**Review by RobinKnipe — CHANGES_REQUESTED** [review]
Just bump the version in drone and you should be good to go! :+1:

File: package.json (hunk 1 above)
Comment (aarongill1) [reply package.json:3]: All done

**Review by RobinKnipe — APPROVED** [review]
LGTM

**Pull request merged**